### ISSUE TITLE
@react-native-community/react-native-device-info: NativeModule.RNDeviceInfo is null

### ISSUE THREAD

**Issue by Hoai-Phong** [react-native-device-info/react-native-device-info#1304]

**Comment by Hoai-Phong** [comment]
I also got the same error , has anyone dealt with it? Please support me. thank you

![image](https://user-images.githubusercontent.com/83800555/132989844-18008eed-3e88-4195-9cb0-a661ebe48794.png)
![image](https://user-images.githubusercontent.com/83800555/132989852-0fcba75a-03e8-435d-be73-86a8dd025895.png)

**Comment by mikehardy** [comment]
The error message is descriptive. You've not installed the module correctly.

**Comment by Hoai-Phong** [comment]
I've tried over and over again and it's still the same. in previous times i used it still works fine now i try to fix it still not working.

**Comment by mikehardy** [comment]
With apologies, it's still a project-specific integration failure. "My native module is null" is always a project-specific integration failure. It's the scourge of all react-native issues lists, if you search here you'll see our issues list is full of them. There's nothing we can do. Triple check your integration steps.

**Comment by Hoai-Phong** [comment]
Thanks for the description , thanks let me try again .
